networks/__init__.py

```diff
@@ -0,0 +1,12 @@
+import torch.nn as nn
+
+from networks.caml import CAML
+from networks.kim_cnn import KimCNN
+from networks.xml_cnn import XMLCNN
+
+
+def get_weight_init_func(config):
+    def init_weights(m):
+        if isinstance(m, nn.Linear) or isinstance(m, nn.Conv2d):
+            getattr(nn.init, config.weight_init + '_')(m.weight)
+    return init_weights
```

Comment:
why don't we just call this fun get_init_weights()??

Reply:
The behavior of the function depends on `config`, so we need a function generator.